docs/developer/taxes.mdx

```diff
@@ -0,0 +1,356 @@
+
+```graphql
+mutation {
+  taxClassCreate(
+    input: {
+      name: "Healthcare products"
+      createCountryRates: [{ countryCode: PL, rate: 8 }]
+    }
+  ) {
+    errors {
+      field
+      message
+    }
+    taxClass {
+      id
+      countries {
+        rate
+      }
+    }
+  }
+}
+```
+
+To assign the tax class to a product, use the following mutation:
+
+```graphql
+mutation {
+  productUpdate(id: "UHJvZHVjdDox", input: { taxClass: "VGF4Q2xhc3M6Mg==" }) {
+    errors {
+      field
+      message
+    }
+    product {
+      taxClass {
+        name
+      }
+    }
+  }
+}
+```
+
+Now, whenever the product is added to checkout, Saleor would use the 8% tax rate.
+
+<aside>
+💡 Note: You can assign tax classes to products directly, or you can assign them to product types using the `productTypeUpdate` mutation. When a product type has a tax class assigned, all products of this type will use it, as long as they don’t have their own tax rates assigned.
+
+</aside>
+
+### Assigning tax rate to a shipping method
+
+To calculate taxes on shipping, you can either configure [the default tax rate for a country](developer/taxes#creating-a-default-country-rate) or create a custom tax class and assign it to the shipping method. Here is an example:
+
+```graphql
+mutation {
+  shippingPriceUpdate(
+    id: "U2hpcHBpbmdNZXRob2RUeXBlOjE="
+    input: { taxClass: "VGF4Q2xhc3M6Mg==" }
+  ) {
+    errors {
+      field
+      message
+    }
+    shippingMethod {
+      taxClass {
+        name
+      }
+    }
+  }
+}
+```
+
+### Querying products with taxed prices for specific country
+
+When using flat rates, the API can list products with prices that include tax rates for different countries. To do this, pass the country code for which you have flat rate configurations available. If the country argument is not specified, the default country for the channel is assumed. (See [Channel.defaultCountry](api-reference/objects/channel.mdx).)
+
+In this example, we are fetching a product list with prices for Poland.
+
+```graphql
+{
+  products(channel: "default-channel", first: 20) {
+    edges {
+      node {
+        id
+        name
+        pricing(address: { country: PL }) {
+          priceRange {
+            start {
+              currency
+              gross {
+                amount
+              }
+              net {
+                amount
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+## Dynamic Taxes
+
+Dynamic Taxes let you delegate the tax calculation to external apps, including 3rd-party tax providers like Avalara or TaxJar or entirely custom tax apps. This means that, instead of setting up static tax rates, your store can use real-time tax rates from external sources. This method is useful when your store operates in multiple countries or regions with complex tax rules or when you want to automate the tax calculation process.
```

Comment:
```suggestion
Dynamic Taxes let you delegate the tax calculation to external apps, including 3rd-party tax providers like Avalara, TaxJar, or custom tax apps. This means that your store can use real-time tax rates from external sources instead of setting up static tax rates. This method is useful when your store operates in multiple countries or regions with complex tax rules or when you want to automate the tax calculation process.
```